core/src/main/res/values/strings.xml

```diff
@@ -133,6 +133,7 @@
   <string name="external_storage">External</string>
   <string name="yes">Yes</string>
   <string name="no">No</string>
+  <string name="confirm_open_in_new_tab">Are you sure you want to open link in new tab?</string>
```

Comment:
The copy makes you think you're doing something wrong by opening the link in a new tab. 

> Do you want to open the link in a new tab?

imo sounds better

Reply:
Done. @rashiq , please take a look at it now.